lib/scheduler/constant_arrival_rate.go

```diff
@@ -0,0 +1,73 @@
+package scheduler
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/loadimpact/k6/lib/types"
+	null "gopkg.in/guregu/null.v3"
+)
+
+const constantArrivalRateType = "constant-arrival-rate"
+
+// ConstantArrivalRateConfig stores config for the constant arrival-rate scheduler
+type ConstantArrivalRateConfig struct {
+	BaseConfig
+	Rate     null.Int           `json:"rate"`
+	TimeUnit types.NullDuration `json:"timeUnit"` //TODO: rename to something else?
```

Comment:
What is this for

Reply:
It's the time unit of the rate, with a default value of `1s`. It's intended as a convenience feature - users don't have to change the value if they want a rate in terms of iterations (`rate`) *per second* (`timeUnit`), but if they want iterations per minute, they can just set `1m` there instead of dividing the `rate` by 60.